libbuteosyncfw/profile/ProfileManager.cpp

```diff
@@ -1031,13 +1031,13 @@ bool ProfileManagerPrivate::parseFile(const QString &aPath, QDomDocument &aDoc)
 
 QDomDocument ProfileManagerPrivate::constructProfileDocument(const Profile &aProfile)
 {
-    //FUNCTION_CALL_TRACE;
+    //FUNCTION_CALL_TRACE(lcButeoTrace);
```

Comment:
hmm, seems to be a lot of them in this file.  I wonder why..?

Reply:
Just a guess / personal experience with similar stuff elsewhere: in order to figure out something, first use search-and-replace to add such logging at every function start; then discover that some functions bring down signal / noise ratio way too much -> comment those out; when finished, forget to think about it and do a proper cleanup before merging / think it might still be needed some day -> you end up with having a semi-random looking set of functions with commented out debug logging ...